src/codesquad/sunny/race/MonsterRace.java

```diff
@@ -0,0 +1,81 @@
+package codesquad.sunny.race;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Random;
+
+public class MonsterRace {
+    private BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+    private int monsterCount;
+    private int attemptCount;
+    private final int count = 5;
+
+    public MonsterRace() throws Exception {
+        showTitle();
+        askMonsterQuantity(count);
+        askAttemptCount(count);
+        racePrint();
+        br.close();
+    }
+
+    private void showTitle() {
+        System.out.println("<스릴만점 건전한 몬스터 경주>");
+    }
+
+    private void askMonsterQuantity(int count) throws Exception {
+        if (count == 0) throw new Exception("시도횟수 초과");
+        System.out.println("몬스터는 모두 몇 마리인가요?");
+        try {
+            monsterCount = Integer.parseInt(br.readLine());
+            checkNegativeCount(monsterCount);
+        } catch (IOException | NegativeNumberException e) {
+            askMonsterQuantity(count - 1);
+        }
+    }
+
+    private void askAttemptCount(int count) throws Exception {
+        if (count == 0) throw new Exception("시도횟수 초과");
+        System.out.println("시도할 회수는 몇 회 인가요?");
+        try {
+            attemptCount = Integer.parseInt(br.readLine());
+            checkNegativeCount(attemptCount);
+        } catch (IOException | NegativeNumberException e) {
+            askAttemptCount(count - 1);
+        }
+    }
+
+    private void checkNegativeCount(int count) throws NegativeNumberException {
+        if (count < 0 || count == 0) {
+          throw new NegativeNumberException();
+        }
+    }
+
+    private int makeRandomCount() {
+        Random random = new Random();
```

Comment:
JAVA 10부터 아래와 같은 문법을 지원합니다.

```suggestion
        val random = new Random();
```

Reply:
저희 수업이 JAVA 8로 통일한다고 하니 Random에 대해서 JAVA 8은 어떻게 사용하는지 확인하겠습니다.